ibc_data/contrasts/ItemRecognition.csv

```diff
@@ -0,0 +1,15 @@
+condition,encode5-encode1,probe5_mem-probe1_mem,probe5_new-probe1_new

```

Comment:
We only have defined those 3 contrasts, but I wonder if it's correct since there are many conditions not considered.

Reply:
You're right, but we only have these three contrasts defined. Maybe we should remove some of the conditions.  Could you open a separate issue for that ? Thx

Reply:
The issue is open!